app/libs/stats.py

```diff
@@ -1,33 +1,295 @@
 from peewee import fn, SQL
-from app.models import Comment, RejectedComment
+from app.models import Comment, RejectedComment, Member, groups, Asset, PendingComment, rejection_reasons
+import re
+import arrow
+from datetime import datetime
+import inspect
+import sys
+import types
 
 
-def total_comments():
-    approved = Comment.select(Comment.id).count()
-    rejected = RejectedComment.select(RejectedComment.id).count()
-    return {'approved': approved,
-            'rejected': rejected,
-            'total': approved + rejected}
+# List of all stats func name
+stats_func_list = ['curr_month_top_commenters',
```

Comment:
@pramitrn we can directly add function here
`stats_func_list = [curr_month_top_commenters, curr_week_top_commenters]`